css/style.css

```diff
@@ -171,29 +171,95 @@ header {
 
 /******************* ABOUT ***********************/
 
-.about {
```

Comment:
Use these initial styles only which looks a bit more appealing and standard across the site.